Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Due to package verification, the latest version below is the unpublished version and the date is meaningless.
 however, it has to be formatted properly to pass verification tests.
 
+## [Unreleased]
+
+### Added
+
+- Added support for DualSense gamepads on desktops (`DualSenseGamepadHID` layout).
```

Comment:
Might be useful to include PS5 in the name here. Just for extra clarity. Maybe

```
- Added support for PS5 DualSense controllers on Mac and Windows.
```

Reply:
fixed